src/OpenTelemetry.Api.ProviderBuilderExtensions/README.md

```diff
@@ -0,0 +1,27 @@
+# OpenTelemetry.Api.ProviderBuilderExtensions
+
+[![NuGet](https://img.shields.io/nuget/v/OpenTelemetry.Api.ProviderBuilderExtensions.svg)](https://www.nuget.org/packages/OpenTelemetry.Api.ProviderBuilderExtensions)
+[![NuGet](https://img.shields.io/nuget/dt/OpenTelemetry.Api.ProviderBuilderExtensions.svg)](https://www.nuget.org/packages/OpenTelemetry.Api.ProviderBuilderExtensions)
+
+## Installation
+
+```shell
+dotnet add package --prerelease OpenTelemetry.Api.ProviderBuilderExtensions
+```
+
+## Overview
+
+The `OpenTelemetry.Api.ProviderBuilderExtensions` package provides extension
+methods and helpers for building `TracerProvider`s and `MeterProvider`s using
+the `Microsoft.Extensions.DependencyInjection` API (primarily
+[IServiceCollection](https://learn.microsoft.com/dotnet/api/microsoft.extensions.dependencyinjection.iservicecollection)).
+
+The `OpenTelemetry.Api.ProviderBuilderExtensions` package is intended for
+instrumentation library authors who need to integrate with the OpenTelemetry SDK
```

Comment:
"who need to provide DI based configuration, but without taking OTel SDK dependency.. "

Feel free to grab this/similar, in a separate PR if needed.